notebooks/geoserver.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.            resp = requests.request(url=signin_url, method="POST", data=data, timeout=10, verify=VERIFY_SSL)
</pre><p><code>headers=HEADERS</code> missing </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>It works without the default <code>HEADERS</code> which specifies json options.</p><p> </p><p>I would leave it as it is, but if you prefer headers to be specified, I would have to use custom headers here, different than the defaults, since the current code will produce an error if using json headers here. </p>

Reply:
I prefer that the notebook specifies the headers explicitly so that it doesn't assume this default will remain valid.
If it states what it wants, the response should always be the same, even if the default changes.